Model/Service/InvoiceService.php

```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Bolt\Boltpay\Model\Service;
+
+use Bolt\Boltpay\Helper\Bugsnag;
+use Magento\Sales\Api\Data\OrderInterface;
+
+class InvoiceService extends \Magento\Sales\Model\Service\InvoiceService
```

Comment:
can you add comment explaining why you need to override this?

Reply:
Sorry for the confusion. It's overriding the Magento core class. It is extending the Magento core class to reuse the Magento core method and property for our new method named prepareInvoiceWithoutItems()